src/woo_publications/metadata/organisation_sync.py

```diff
@@ -0,0 +1,99 @@
+from io import StringIO
+from pathlib import Path
+
+from django.core.management import call_command
+
+import requests
+from glom import PathAccessError, T, glom
+
+from woo_publications.metadata.constants import OrganisationOrigins
+from woo_publications.metadata.models import Organisation
+
+MUNICIPALITY_WAARDENLIJST_URL = "https://repository.officiele-overheidspublicaties.nl/waardelijsten/rwc_gemeenten_compleet/4/json/rwc_gemeenten_compleet_4.json"
+SO_WAARDENLIJST_URL = "https://repository.officiele-overheidspublicaties.nl/waardelijsten/rwc_samenwerkingsorganisaties_compleet/1/json/rwc_samenwerkingsorganisaties_compleet_1.json"
+OORG_WAARDENLIJST_URL = "https://repository.officiele-overheidspublicaties.nl/waardelijsten/rwc_overige_overheidsorganisaties_compleet/8/json/rwc_overige_overheidsorganisaties_compleet_8.json"
+
+MUNICIPALITY_WAARDENLIJST_TYPE = "https://identifier.overheid.nl/tooi/def/ont/Gemeente"
+SO_WAARDENLIJST_TYPE = (
+    "https://identifier.overheid.nl/tooi/def/ont/Samenwerkingsorganisatie"
+)
+OORG_WAARDENLIJST_TYPE = (
+    "https://identifier.overheid.nl/tooi/def/ont/Overheidsorganisatie"
+)
+WAARDENLIJST_URLS = [
+    MUNICIPALITY_WAARDENLIJST_URL,
+    SO_WAARDENLIJST_URL,
+    OORG_WAARDENLIJST_URL,
+]
+
+TYPE_MAPPING = [
+    (
+        MUNICIPALITY_WAARDENLIJST_URL,
+        MUNICIPALITY_WAARDENLIJST_TYPE,
+        OrganisationOrigins.municipality_list,
+    ),
+    (SO_WAARDENLIJST_URL, SO_WAARDENLIJST_TYPE, OrganisationOrigins.so_list),
+    (OORG_WAARDENLIJST_URL, OORG_WAARDENLIJST_TYPE, OrganisationOrigins.oorg_list),
+]
+
+SPEC = {
+    "identifier": "@id",
+    "naam": T["http://www.w3.org/2000/01/rdf-schema#label"][0]["@value"],
+}
+
+
+class OrganisatieWaardenlijstError(Exception):
+    def __init__(self, message: str):
+        self.message = message
+        super().__init__(message)
+
+
+def update_organisation(file_path: Path):
+    for waardenlijst_url, waardenlijst_type, oorsprong in TYPE_MAPPING:
```

Comment:
I choose to loop over all the url endpoints because from what I can read I assume the wanted result is that we make 1 fixture for all the organisations instead of 3 fixtures based on the origin type. Is this correct or do is the wanted usecase that we create 3 fixture files?

Reply:
Indeed - a single magical fixture to rule them all!